tests/OSGPKIUtilsTests.py

```diff
@@ -18,26 +18,25 @@ class OSGPKIUtilsTests(unittest.TestCase):
     def test_csr_generation(self):
         '''Generate a basic CSR'''
         cert = OSGPKIUtils.Cert(self.FQDN, email=EMAIL)
-        self.assert_(cert.x509request, 'missing CSR contents')
+        self.assertTrue(cert.x509request, 'missing CSR contents')
 
     def test_alt_name_csr_generation(self):
         '''Generate a CSR with multiple SANs'''
         alias = 'test-san.' + DOMAIN
         cert = OSGPKIUtils.Cert(self.FQDN, altnames=[alias], email=EMAIL)
         csr_contents = cert.x509request.as_text()
-        self.assert_(re.search(r'X509v3 Subject Alternative Name: critical', csr_contents),
+        self.assertIsNone(re.search(r'X509v3 Subject Alternative Name: critical', csr_contents),
```

Comment:
This seems to flip the test around. We want to make sure that we find this search string